godot-core/src/builtin/callable.rs

```diff
@@ -392,10 +393,21 @@ mod custom_callable {
     ) {
         let arg_refs: &[&Variant] = Variant::borrow_ref_slice(p_args, p_argument_count as usize);
 
-        let c: &mut C = CallableUserdata::inner_from_raw(callable_userdata);
+        let name = {
+            let c: &C = CallableUserdata::inner_from_raw(callable_userdata);
+            c.to_string()
+        };
+        let ctx = CallContext {
+            class_name: Cow::Borrowed("<Callable>"),
+            function_name: name.as_str(),
+        };
 
-        let result = c.invoke(arg_refs);
-        crate::meta::varcall_return_checked(result, r_return, r_error);
+        crate::private::handle_varcall_panic(&ctx, &mut *r_error, move || {
+            let c: &mut C = CallableUserdata::inner_from_raw(callable_userdata);
```

Comment:
`inner_from_raw` is now invoked twice, while it was only once before. It also makes sure that `callable_userdata` is captured and not the reference, which I guess is a good thing.

Could you maybe add a comment describing why it's called twice?